concepts/05 Widgets/DropDownBox/15 Synchronize with the Embedded Element.md

```diff
@@ -5,12 +5,12 @@ The **DropDownBox**'s and embedded widget's data sources can be the same or diff
 
         <!--JavaScript-->
         // Different data sources, both have the ID field
-        var widgetData = [
+        const widgetData = [
```

Comment:
Please, pay attention, that snippets on this topic are very synthetic and not full. Should we copy-paste the data from the previous topic and make this one more alive?

Reply:
I guess that the point of this reduction is to make this snippet a bit easier to understand. Users can review the working implementation if they follow one of the links to demos below.

Reply:
Ok, thx